python/ray/data/dataset.py

```diff
@@ -405,17 +405,6 @@ def map_batches(
         To learn more about writing functions for :meth:`~Dataset.map_batches`, read
         :ref:`writing user-defined functions <transform_datasets_writing_udfs>`.
 
-        .. tip::
```

Comment:
Remove overly spammy / incorrect tips

Reply:
Why it's incorrect though? We no longer want to move usage to higher-level APIs like preprocessor ?

Reply:
It's kind of the opposite of https://github.com/ray-project/enhancements/pull/25.

I don't think this was ever correct in the first place. Also, it's not appropriate for Data users to be told to use preprocessor when they may not need it.